src/BenchmarkDotNet/Attributes/Jobs/LegacyJitX86JobAttribute.cs

```diff
@@ -1,7 +1,9 @@
-﻿using BenchmarkDotNet.Jobs;
+﻿using System;
+using BenchmarkDotNet.Jobs;
 
 namespace BenchmarkDotNet.Attributes
 {
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true)]
```

Comment:
`AllowMultiple` won't be useful for this type: it does not allow for any kind of customizations
```suggestion
```